Sources/OtlpGRPCSpanExporting/LogRecordExporter.swift

```diff
@@ -0,0 +1,66 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift OpenTelemetry open source project
+//
+// Copyright (c) 2023 Moritz Lang and the Swift OpenTelemetry project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import struct Foundation.Data
+import class Foundation.ProcessInfo
+import struct Foundation.URL
+import GRPC
+import Logging
+import NIO
+import OpenTelemetry
+
+public final class OtlpGRPCLogRecordExporter: OTelLogRecordExporter {
+    private let client: Opentelemetry_Proto_Collector_Logs_V1_LogsServiceNIOClient
+    private let logger: Logger
+
+    public init(config: OtlpGRPCSpanExporter.Config) {
+        let channel = ClientConnection
+            .insecure(group: config.eventLoopGroup)
+            .connect(host: config.host, port: config.port)
+
+        self.client = Opentelemetry_Proto_Collector_Logs_V1_LogsServiceNIOClient(
+            channel: channel,
+            defaultCallOptions: .init(timeLimit: .timeout(.seconds(10)))
+        )
+        self.logger = Logger(
+            label: "OtlpGRPCLogRecordExporter",
+            factory: { label in
```

Comment:
Yep 👍 In general, how do we go about logging from within a log handler? It's unfortunate that these logs don't make it to the OTel backend, but I'm also not sure how to pull that off.

Reply:
sounds a bit off to log from a log handler? do you truly need to do that?

Reply:
How else would I let people know about gRPC failures while exporting the logs?

Reply:
okey but those don't need to make it back into the otel backend since that's what is broken I suppose right? I may be misunderstanding the cycle here?

Reply:
Not necessarily. That's why I'm asking. I basically want to know which backend to use for those logs. Probably stream log handler makes the most sense since it's built in, right?